src/DotNet.Testcontainers/Clients/TestcontainersClient.cs

```diff
@@ -222,5 +222,10 @@ public Task<string> BuildAsync(IImageFromDockerfileConfiguration configuration,
     {
       return this.images.BuildAsync(configuration, ct);
     }
+
+    public Task ExportFilesFromContainerAsync(string id, string containerPath, Stream outStream, CancellationToken ct = default)
```

Comment:
Here is how it was extracted https://github.com/GodelTech/CodeReview.Orchestrator/blob/c070b8960089a183a2bcaa0b3726b5a39c6e9c65/src/CodeReview.Orchestrator/Services/TarArchiveService.cs#L39-L69

Reply:
Don't you need the host path as well? I guess the part that extracts the archive is missing, right (thats your link)?